test/core/channelz/channelz_registry_test.cc

```diff
@@ -168,6 +182,13 @@ TEST(ChannelzRegistry, ThreadStressTest) {
   }
 }
 
+INSTANTIATE_TEST_SUITE_P(ChannelRegistrySuite, ChannelzRegistryTest,
```

Comment:
Suggest moving this up to line 50, so it's right after the test class and therefore a bit easier to find.